<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="laStrokeColor">
       <property name="text">
        <string>Stroke color:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbStrokeColor">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="laStrokeOpacity">
       <property name="text">
        <string>Stroke opacity:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="slStrokeOpacity">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="laStrokeWidth">
       <property name="text">
        <string>Stroke width:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sbStrokeWidth">
       <property name="specialValueText">
        <string>No Stroke</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="laDashType">
       <property name="text">
        <string>Dash type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="laDashOffset">
       <property name="text">
        <string>Dash offset:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="dsbDashOffset"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="laLineCapStyle">
       <property name="text">
        <string>Line cap style:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="squareCap">
         <property name="text">
          <string>Square</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgCapStyle</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="buttCap">
         <property name="text">
          <string>Flat</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgCapStyle</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="roundCap">
         <property name="text">
          <string>Round</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgCapStyle</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="laLineJoinStyle">
       <property name="text">
        <string>Line join style:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="bevelJoin">
         <property name="text">
          <string>Bevel</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgJoinStyle</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="miterJoin">
         <property name="text">
          <string>Miter</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgJoinStyle</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="roundJoin">
         <property name="text">
          <string>Round</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgJoinStyle</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="laStrokeMiterLimit">
       <property name="text">
        <string>Stroke miter limit:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="dsbStrokeMiterLimit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFill">
     <property name="title">
      <string>Fill property</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="laFillColor">
        <property name="text">
         <string>Fill color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbFillColor">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="laFillOpacity">
        <property name="text">
         <string>Fill opacity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="slFillOpacity">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>miterJoin</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsbStrokeMiterLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgCapStyle"/>
  <buttongroup name="bgJoinStyle"/>
 </buttongroups>
</ui>
